pkg/generator/appconfiguration/generator/accessories/database/database_generator.go

```diff
@@ -98,19 +98,27 @@ func (g *databaseGenerator) Generate(spec *models.Spec) error {
 func (g *databaseGenerator) injectSecret(secret *v1.Secret) error {
 	// Inject the database information into the containers of service workload.
 	if g.workload.Service != nil {
```

Comment:
What is the difference between Service and Job? Can we combine them into one code block?